lib/ButtonNew/ButtonIcon/ButtonIconContained.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { ButtonIcon } from 'lib';
+import { sizes } from '../common';
 
-function ButtonIconContained(props) {
-  const classNames = `button-icon-contained ${props.className}`;
+function ButtonIconContained({ className, ...rest }) {
```

Comment:
If we dont do this and pass a className to this component it overrides the className we set on the line below